app/models/categorisation.rb

```diff
@@ -0,0 +1,9 @@
+class Categorisation < ActiveRecord::Base
```

Comment:
Maybe `categorisation` is British spelling and `categorization` is American spelling.